Library/Formula/gxml.rb

```diff
@@ -0,0 +1,73 @@
+class Gxml < Formula
+  desc "GObject-based XML DOM API"
+  homepage "https://wiki.gnome.org/GXml"
+  url "https://download.gnome.org/sources/gxml/0.6/gxml-0.6.0.tar.xz"
+  sha256 "50f5e1b691fcc2994264565bcbfa4cfcd386bd01ea98c9fbec3ae4ec66ce7fc8"
+
+  depends_on "pkg-config" => :build
+  depends_on "intltool" => :build
+  depends_on "vala" => :build
+  # gnu-sed dependency to be removed when 0.6.1 is released
+  # usage here circumvents a bug in gxml when using Mac OS X's sed
+  depends_on "gnu-sed" => :build
+  depends_on "libxml2"
+  depends_on "glib"
+  depends_on "libgee"
+  depends_on "gobject-introspection"
+
+  def install
+    # ensures that the gobject-introspection files remain within the keg
+    inreplace "gxml/Makefile.in", "@HAVE_INTROSPECTION_TRUE@girdir = $(INTROSPECTION_GIRDIR)",
```

Comment:
Can you fix this to use the `inreplace "gxml/Makefile.in" do |s|` syntax highlighted in the other thread? Thanks!